design-documents/graph-ql/coverage/Cart.graphqls

```diff
@@ -57,3 +59,8 @@ type CheckoutPaymentMethod {
 type CartGiftCard {
     code: String!
 }
+
+type CartError {
```

Comment:
Please explain in proposal why it is not possible to use built-in mechanism for GraphQL errors along with `\Magento\Framework\Validation\ValidationException::getErrors`

Reply:
The main reason that resolvers have conditions to prevent the execution of add product to cart and show items in the cart(example app/code/Magento/QuoteGraphQl/Model/Cart/AddProductsToCart.php). That behavior difference with frontend where the customer could see products in cart with messages on frontend but in graphql query he got the error.